docs/developing/auth.md

```diff
@@ -95,15 +95,49 @@ the
 [sandstorm-rawapi-example](https://github.com/sandstorm-io/sandstorm-rawapi-example)
 repository on GitHub.
 
-## Further reading
 
-You might be interested in looking at:
+## Defining permissions and roles
+
+Apps define permissions by providing a
```

Comment:
IMHO many app authors won't know if they're using `sandstorm-http-bridge` or what a `UIView` is. IMHO we run the risk of sounding like git's documentation, where every man page is great advice if you know git's internals and not that useful if you don't know git's internals.

Given that, and given that the second and third paragraphs are crystal-clear to someone with minimal understanding of Sandstorm, I propose that you move this paragraph to after the "From Sandstorm's perspective..." paragraph.

I also suggest mentioning `vagrant-spk` since more app authors are likely to have seen that than `sandstorm-http-bridge`. For example, replace:

> Apps that use `sandstorm-http-bridge` can specify a `ViewInfo` value in the `PackageDefinition.bridgeConfig` field of their `sandstorm-pkgdef.capnp`.

with:

> Most apps, such as those created with `vagrant-spk`, use `sandstorm-http-brige` and can specify a `ViewInfo` value in the `PackageDefinition.bridgeConfig` field of their `sandstorm-pkgdef.capnp`.

It's also too bad (IMHO) that the capnp comments you link to include a many-line essay about confused deputies and membranes, both of which are terms with which most app authors will be unfamiliar, before the part that app authors would find useful.  For that reason, I suggest de-link-ifying `UiView.ViewInfo` here, and adding a link to it below in a **Further reading** section, underneath the examples.